### PR TITLE
Update GCP doc to add deprecation note for logging and monitoring

### PR DESCRIPTION
**Pull request by sharvarikhedkar**
This PR adds a note to documentation - [Chapter 9: Logging and Monitoring ](https://dxp-docp-prod.apps.ext-waf.spoke.prod.us-west-2.aws.paas.redhat.com/documentation/en-us/ansible_on_clouds/2.4/html-single/red_hat_ansible_automation_platform_from_gcp_marketplace_guide/index?lb_target=preview&check_logged_in=1#assembly-gcp-monitoring-logging) regarding logging and monitoring deprecation in future release. 

Jira: https://issues.redhat.com/browse/AAP-15317

### CODE REVIEW

**Review by chrisahl — CHANGES_REQUESTED** [review]
Doc changes for the next release need to be against branch `aap-clouds-next`.  Using `aap-clouds-latest` would cause it to go live into the current release docs once merged.

@ianf77 Are you ok with the text being added to indicate a feature we will deprecate in a future release?

File: stories/assembly-gcp-monitoring-logging.adoc
Comment (ianf77) [inline stories/assembly-gcp-monitoring-logging.adoc:10]: You should also add something to the Release  notes appendix.

**Review by ianf77 — APPROVED** [review]
One comment, which doesn't need to be actioned.

File: stories/assembly-gcp-monitoring-logging.adoc
Comment (sharvarikhedkar) [reply stories/assembly-gcp-monitoring-logging.adoc:10]: Should I add a deprecation note under `Deprecated and removed features` section in[ release notes](https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html-single/red_hat_ansible_automation_platform_from_gcp_marketplace_guide/index#assembly-appendix-aap-release-notes)?

**Comment by chrisahl** [comment]
When this PR is ready, do not merge into aap-clouds-latest, it is for the next release, aap-clouds-next.  So the PR needs to change the target branch

**Comment by sharvarikhedkar** [comment]
@chrisahl changed the target branch to `aap-clouds-next`.

**Review by chrisahl — APPROVED** [review]
Looks good to me.  Thanks.

**Pull request merged**
